experiments/README.md

```diff
@@ -0,0 +1,48 @@
+# Toxic Classification Framework
+
+This directory contains a framework and a range of models for the classification of toxic comments and other attributes of online conversation. We have structured the framework as a series of common files and templates to quickly construct models on top of the Keras and tf.estimator APIs. The templates also demonstrate how these models can be trained using [Google ML Engine](https://cloud.google.com/ml-engine/) and evaluated using comet.ml. 
+
+## Environment Setup
+
+1.  Setup a (virtualenv)[https://virtualenvwrapper.readthedocs.io/en/latest/] for
+    the project (recommended, but technically optional).
+
+    Python 2:
+
+    ```
+    python -m virtualenv env
+    ```
+
+    Python 3:
+
+    ```
+    python3 -m venv env
+    ```
+
+    From either to enter your virtual env:
+
+    ```shell
+    source env/bin/activate
+    ```
+
+2.  Install library dependencies:
+
+    ```shell
+    pip install -r requirements.txt
+    ```
+
+## Training an Existing Model
+
+To train an existing model,  simply execute the command `./tf_traner/MODEL_NAME/run.sh` (to run locally) or the command `./tf_trainer/MODEL_NAME/run_ml_engine.sh` to run training on [Google ML Engine](https://cloud.google.com/ml-engine/).
```

Comment:
I agree, but I think we should still have a place where we explain what is happening in these files in more depth. I don't want a new user of our library/framework to have to absorb all that is happening in those 5 files before building a new model. Do you think it would be better to add a more detailed README to the directory of each model explaining the role of each file?